lib/e2e/Cargo.toml

```diff
@@ -17,6 +17,7 @@ once_cell.workspace = true
 koba.workspace = true
 e2e-proc = { path = "../e2e-proc" }
 toml = "0.8.13"
+serde_json = "1.0.119"
```

Comment:
Do we need this? I can't see it used anywhere.

Reply:
no need. Artifact of the previous approach